src/Error/Error.php

```diff
@@ -111,6 +115,8 @@ private function updateRepr(): void
             $this->file = $this->source->getPath();
             if ($this->lineno > 0) {
                 $this->line = $this->lineno;
+            } else {
+                $this->line = -1;
```

Comment:
Not related, but let's not keep the PHP line if we override the file with a Twig one.